### PR TITLE
Review pull request

### PR DESCRIPTION
**Pull request by NoszDom**
SonarLint error fixes.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +34,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (Feketerig) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:37]: Good job.

**Review by Feketerig — COMMENTED** [review]
Done review.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,8 +14,12 @@
 
   private int torpedoCount = 0;
 
+  private Random generator;
```
[the remark this answers is not in the document]
Comment (NoszDom) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: Thanks.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (NoszDom) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:37]: Thank you.

**Review by Feketerig — APPROVED** [review]

**Comment by Feketerig** [comment]
Nice job.

**Pull request merged**